internal/app/activity/activity_handler.go

```diff
@@ -132,6 +132,7 @@ func (a *activityHandler) ListActivities(filter *entity.ActivityFilter, options
 }
 
 func (a *activityHandler) CreateActivity(activity *entity.Activity) (*entity.Activity, error) {
+	activity.CreatedBy = "Creator"
```

Comment:
I guess that's a placeholder. Based on the comment I made we need to decide if that should really be a freetext field or rather a foreign key reference to a User